### ISSUE TITLE
training struck with Start training from epoch: 0, iter: 0

### ISSUE THREAD

**Issue by nowfalcodmeric** [xinntao/Real-ESRGAN#236]
2022-01-28 12:00:05,326 INFO: Loading: params_ema does not exist, use params.
2022-01-28 12:00:05,327 INFO: Loading RRDBNet model from experiments/pretrained_models/ESRGAN_SRx4_DF2KOST_official-ff704c30.pth, with param key: [params].
2022-01-28 12:00:05,409 INFO: Use Exponential Moving Average with decay: 0.999
2022-01-28 12:00:05,579 INFO: Network [RRDBNet] is created.
2022-01-28 12:00:05,629 INFO: Loading: params_ema does not exist, use params.
2022-01-28 12:00:05,629 INFO: Loading RRDBNet model from experiments/pretrained_models/ESRGAN_SRx4_DF2KOST_official-ff704c30.pth, with param key: [params].
2022-01-28 12:00:05,709 INFO: Loss [L1Loss] is created.
2022-01-28 12:00:05,722 INFO: Model [RealESRNetModel] is created.
2022-01-28 12:00:05,814 INFO: Start training from epoch: 0, iter: 0

**Comment by kodxana** [comment]
What GPU and Batch size?

**Comment by Steven-vanb** [comment]
Had this same issue, lowering the batch size seems to fix it

**Comment by 873626104** [comment]
这种我出现过好多次，好像是因为没有正常结束强行转换早晨造成的，用了重装环境大法才解决
